portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,46 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Home | Peter Geertsema's Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Roboto&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    <div id="nav-bar">
+      <ul id="nav-list">
+        <li class="nav-item"><a class="nav-active" href="index.html">Home</a></li>
+        <li class="nav-item"><a class="nav-inactive" href="mickey.html">Mickey</a></li>
+        <li class="nav-item"><a class="nav-inactive" href="projects.html">Projects</a></li>
+      </ul>
+    </div>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>About Me</h1>
+      <p>My name is Peter Geertsema. I'm a rising Sophomore at the University of Maryland - College Park where I study Computer Science and Economics. Go Terps!</p>
```

Comment:
Although Google doesn't require it for HTML(https://engdoc.corp.google.com/eng/doc/devguide/htmlcss/styleguide.md?cl=head#HTML_Line-Wrapping), I would prefer it, so that all code is visible within github preview without scrolling back and forth. Could you please reformat this and other files so that everything that can be line wrapped, is?

Reply:
OK - I used a soft rule of 100 chars per line. 
Is 100 chars good or should I plan on using less?

Reply:
100 is perfect!